front/src/Phaser/Game/GameMap.ts

```diff
@@ -120,6 +114,30 @@ export class GameMap {
         return [];
     }
 
+    public getCollisionsGrid(): number[][] {
+        const collisionsLayer = this.findPhaserLayer("collisions");
+        if (!collisionsLayer) {
+            return [];
+        }
+        const grid: number[][] = [];
+        for (let y = 0; y < collisionsLayer.height; y += 1) {
+            const row: number[] = [];
+            for (let x = 0; x < collisionsLayer.width; x += 1) {
+                row.push(collisionsLayer.getTileAt(x, y) ? 1 : 0);
+            }
+            grid.push(row);
+        }
+        return grid;
+    }
```

Comment:
Most of our maps are developed with a layer called "collisions" but it is only a convention.

What makes a tile collidable is the fact that it the tile in the tileset has the "collides" property set.

So I think you should rather construct a list of all the tiles that are collidable, and for each tile of each layer, see if the tile is belonging to the list of collidable tiles.